examples/flux-source/index.ts

```diff
@@ -0,0 +1,248 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as kubernetes from "@pulumi/kubernetes";
+import { ConfigGroup } from "@pulumi/kubernetes/yaml";
+import * as flux from "@worawat/flux";
+
+const config = new pulumi.Config();
+const deployNamespace = config.get("namespace") || 'default';
+const deployNamespaceList = config.getObject<string[]>("namespaces") || [deployNamespace];
+const operatorVersion = config.get("operator-version") || "v1.9.0";
+const crdVersion = config.get("crd-version") || "v1.9.0";
+
+// -- Flux installation
+
+const fluxManifests = flux.getFluxInstallOutput({
+    targetPath: "local",
+    components: ['source-controller'],
+    networkPolicy: true,
+});
+
+// this effectively installs the Flux controllers
+const fluxGroup = fluxManifests.content.apply(c => new ConfigGroup("flux-install", { yaml: c }));
```

Comment:
```suggestion
const fluxGroup = new ConfigGroup("flux-install", { yaml: fluxManifests.content });
```

Reply:
Did you try this? It complains that `.yaml` is typed `string` not `Output<string>`, to me.